### PR TITLE
[Mux/Merge] clear collect pad data

### PR DESCRIPTION
**Pull request by jaeyun-jung**
Clear old buffer in collect pad data when received flush event.

Signed-off-by: Jaeyun <jy1210.jung@samsung.com>

### CODE REVIEW

File: gst/nnstreamer/tensor_merge/gsttensormerge.c [hunk 1]
```diff
@@ -355,6 +356,8 @@ gst_tensor_merge_sink_event (GstCollectPads * pads, GstCollectData * data,
   switch (GST_EVENT_TYPE (event)) {
     case GST_EVENT_FLUSH_STOP:
       tensor_merge->need_segment = TRUE;
+      tensor_merge->need_set_time = TRUE;
+      gst_tensor_time_sync_flush (tensor_merge->collect);
```
Comment (dongju-chae) [inline gst/nnstreamer/tensor_merge/gsttensormerge.c:360]: How about checking whether `tensor_merge->collect` is null

File: gst/nnstreamer/tensor_mux/gsttensormux.c [hunk 2]
```diff
@@ -343,6 +344,8 @@ gst_tensor_mux_sink_event (GstCollectPads * pads, GstCollectData * data,
   switch (GST_EVENT_TYPE (event)) {
     case GST_EVENT_FLUSH_STOP:
       tensor_mux->need_segment = TRUE;
+      tensor_mux->need_set_time = TRUE;
+      gst_tensor_time_sync_flush (tensor_mux->collect);
```
Comment (dongju-chae) [inline gst/nnstreamer/tensor_mux/gsttensormux.c:348]: ditto

File: gst/nnstreamer/tensor_merge/gsttensormerge.c (hunk 1 above)
Comment (jaeyun-jung) [reply gst/nnstreamer/tensor_merge/gsttensormerge.c:360]: I added new PR to check null param (not here, in public functions for collect pad)

**Review by dongju-chae — APPROVED** [review]
LGTM

**Review by gichan-jang — APPROVED** [review]

File: gst/nnstreamer/tensor_common_pipeline.c [hunk 3]
```diff
@@ -240,6 +271,12 @@ gst_tensor_time_sync_buffer_from_collectpad (GstCollectPads * collect,
   GstTensorsConfig in_configs;
   GstClockTime base_time = 0;
 
+  g_return_val_if_fail (collect != NULL, FALSE);
+  g_return_val_if_fail (sync != NULL, FALSE);
+  g_return_val_if_fail (tensors_buf != NULL, FALSE);
+  g_return_val_if_fail (configs != NULL, FALSE);
+  g_return_val_if_fail (is_eos != NULL, FALSE);
```
Comment (helloahn) [inline gst/nnstreamer/tensor_common_pipeline.c:278]: :+1:  I like this part.

**Review by helloahn — APPROVED** [review]
LGTM

**Pull request merged**